src/explore-education-statistics-common/src/services/permalinkService.ts

```diff
@@ -37,7 +37,7 @@ export default {
     releaseId?: string,
   ): Promise<Permalink> {
     if (releaseId) {
-      return dataApi.post(`/release/${releaseId}/permalink`, query);
+      return dataApi.post(`/permalink/release/${releaseId}`, query);
```

Comment:
Just out of interest was there something broken here?

Reply:
Yeah, if the releaseId argument was supplied, it was trying to call a non-existent endpoint.  Because all of the PermalinkController methods in the Data API projects are prefixed with `/api/permalink`.